Makefile

```diff
@@ -48,6 +48,8 @@ quality: $(VENV_REQUIREMENT)
 	find * -type f -name '*.sh' -exec shellcheck --external-sources {} +
 	python manage.py makemigrations --check --dry-run --noinput || (echo "⚠ Missing migration ⚠"; exit 1)
 	python manage.py collectstatic --no-input
+	# Make sure pytest help is still accessible.
+	pytest --help >/dev/null 2>&1
```

Comment:
Je dirais plutôt juste `pytest --help 1> /dev/null`.
On évite le wall of text quand ça marche mais on devrait l'avoir quand ça plantera.

Reply:
Deal.